packages/testlab/src/test-sandbox.ts

```diff
@@ -0,0 +1,112 @@
+// Copyright IBM Corp. 2017,2018. All Rights Reserved.
+// Node module: @loopback/testlab
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {tmpdir} from 'os';
+import {createHash} from 'crypto';
+import {resolve, join, parse} from 'path';
+import * as util from 'util';
+import {
+  mkdirSync,
+  existsSync,
+  mkdir as mkdir_,
+  copyFile as copyFile_,
+} from 'fs';
+const promisify = util.promisify || require('util.promisify/implementation');
+
+const rimraf = promisify(require('rimraf'));
+const mkdir = promisify(mkdir_);
+const copyFile = promisify(copyFile_);
```

Comment:
I'd rename these promisified values so that they're explicit about their async nature.
`copyFile -> copyFileAsync`